peachjam/js/components/FindDocuments/index.vue

```diff
@@ -662,11 +662,13 @@ export default {
       const facets = [];
       const fields = this.facets.map(facet => facet.name).concat(['date__range', 'date__gte', 'date__lte']);
 
+      let currentKey = '';
```

Comment:
A better way of doing this is iterating over the keys only, not the key/value pairs. Then you don't need to de-dup like this.